financial-connections/src/main/java/com/stripe/android/financialconnections/ui/InstitutionPickerScreen.kt

```diff
@@ -0,0 +1,152 @@
+@file:OptIn(ExperimentalFoundationApi::class, ExperimentalMaterialApi::class)
+
+package com.stripe.android.financialconnections.ui
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.GridCells
+import androidx.compose.foundation.lazy.LazyVerticalGrid
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Card
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.LinearProgressIndicator
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.airbnb.mvrx.Async
+import com.airbnb.mvrx.Fail
+import com.airbnb.mvrx.Success
+import com.airbnb.mvrx.compose.collectAsState
+import com.airbnb.mvrx.compose.mavericksViewModel
+import com.stripe.android.financialconnections.model.Institution
+import com.stripe.android.financialconnections.model.InstitutionResponse
+import com.stripe.android.financialconnections.presentation.InstitutionPickerViewModel
+
+@Composable
+internal fun InstitutionPickerScreen() {
+    val viewModel: InstitutionPickerViewModel = mavericksViewModel()
+    val state by viewModel.collectAsState()
+    InstitutionPickerContent(
```

Comment:
We should make sure that we can also inject a theme here so that the components have the same styling.

Reply:
Is this intended to be used in the manual connect flow or in mobile elements?

Reply:
There's some work in progress to add theming on top if this prototype - https://github.com/stripe/stripe-android/commit/843b56f177684a0c5a1fac2d0bf6d499291d3920